src/fsharp/infos.fs

```diff
@@ -2968,6 +2968,9 @@ let TrySelectMemberVal g optFilter typ pri _membInfo (vref:ValRef) =
 /// Query the immediate methods of an F# type, not taking into account inherited methods. The optFilter
 /// parameter is an optional name to restrict the set of properties returned.
 let GetImmediateIntrinsicMethInfosOfType (optFilter,ad) g amap m typ =
+    let getFSharpMetadata() =
```

Comment:
Generally this code avoids inner functions - just create a new top level function ``GetImmediateIntrinsicFSharpMethInfosOfType`` please